www/src/i18n/index.js

```diff
@@ -0,0 +1,18 @@
+import arMessages from './messages/ar.json';
+// no need to import en messages-- they are in the defaultMessage field
+import es419Messages from './messages/es_419.json';
+import frMessages from './messages/fr.json';
+import kokrMessages from './messages/ko_kr.json';
+import ptbrMessages from './messages/pt_br.json';
+import zhcnMessages from './messages/zh_CN.json';
+
+const messages = {
```

Comment:
Will this override languages when we release the npm package to the piblic?

Reply:
This is just the gallery.

Reply:
well... specifically, this is generating the intl top-level provider that enables the subcomponents to also have intl capability (which they require)